language/entity-extraction/src/main/java/com/google/cloud/language/automl/entity/extraction/samples/CreateModel.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.language.automl.entity.extraction.samples;
+
+// [START automl_natural_language_entity_create_model]
+
+import com.google.api.gax.longrunning.OperationFuture;
+import com.google.cloud.automl.v1beta1.AutoMlClient;
+import com.google.cloud.automl.v1beta1.LocationName;
+import com.google.cloud.automl.v1beta1.Model;
+import com.google.cloud.automl.v1beta1.OperationMetadata;
+import com.google.cloud.automl.v1beta1.TextExtractionModelMetadata;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+
+class CreateModel {
+
+  // Create a model
+  static void createModel(
+      String projectId, String computeRegion, String datasetId, String modelName)
+      throws IOException, InterruptedException, ExecutionException {
+    // String projectId = "YOUR_PROJECT_ID";
+    // String computeRegion = "YOUR_COMPUTE_REGION";
+    // String datasetId = "YOUR_DATASET_ID";
+    // String modelName = "YOUR_MODEL_NAME";
+
+    // Instantiates a client
+    AutoMlClient client = AutoMlClient.create();
```

Comment:
Should all these files be wrapped in a try block to auto close the client?

Reply:
Will be fixed in latest commit across all samples.